covid19-etl/utils/metadata_helper.py

```diff
@@ -109,6 +109,29 @@ def get_str_latest_submitted_date_idph(self):
             return loc[0]["date"]
         return None
 
+    def get_existing_summary_locations_idph(self):
```

Comment:
you can keep in the ETL file the functions that are specific to that ETL and only used once

Reply:
or you can call it `get_existing_summary_locations`. but it's still only used once so it doesn't need to be in the utils